tests/www_rbac/test_views.py

```diff
@@ -448,13 +448,23 @@ def test_refresh(self):
 
 
 class TestConfigurationView(TestBase):
-    def test_configuration(self):
+    def test_configuration_do_not_expose_config(self):
         self.logout()
         self.login()
+        conf.set("webserver", "expose_config", "False")
```

Comment:
@kaxil This change is not reflected in the configuration. Any ideas on how to address this issue.

Reply:
@phani8996 Because it was checking the wrong condition.

`test_configuration_do_not_expose_config` should have `expose_config=False`, we had it `True`. I have fixed it

Reply:
The tests should pass now

Reply:
My bad, i added it in wrong direction 😅😅